tests/RobotFramework/tests/cumulocity/bridge_config/builtin_bridge.robot

```diff
@@ -10,7 +10,7 @@ Test Teardown       Get Logs
 *** Test Cases ***
 Connection test
     [Documentation]    Repeatedly test the cloud connection
-    FOR    ${attempt}    IN RANGE    0    10    1
+    FOR    ${attempt}    IN RANGE    0    1    1
```

Comment:
Why this change?

Reply:
Accidental change. Reverted in 0a066063d